src/com/franz/sud/components/engine/GameManager.java

```diff
@@ -77,10 +79,9 @@ private void getUserAction() {
 
         switch (opt) {
             case "i":
-                inventoryManager.openInventoryMenu();
+                inventoryService.openInventoryMenu();
                 break;
             case "m":
-                map.showMap();
                 move();
                 break;
             case "c":

```

Comment:
I can't comment on Line 90 so I'm putting it here. Please move the `default` case at the end of the switch.